src/fix/main.c

```diff
@@ -334,6 +419,12 @@ do { \
 				features |= BATTERY;
 				break;
 
+			case 'M':
+			case 'm':
+				tryReadSlice("ULTIRUMBLE");
+				features |= MULTIRUMBLE;
```

Comment:
```suggestion
				// MULTIRUMBLE implies RUMBLE (used for TPP1)
				features |= MULTIRUMBLE | RUMBLE;
```

Reply:
Features merely indicate which is requested, but `MULTIRUMBLE` enables both bits 0 and 1 further below. This is so that e.g. a warning can be produced for requesting both.